charts/yandex-csi-controller/templates/storage-classes.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: yc-network-hdd
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
```

Comment:
That seems like a candidate for parametrization. With a "default" value set to `yc-network-hdd`.

Reply:
i will fix it